merlin-server/constraints-dsl-compiler/src/libs/constraints-edsl-fluent-api.ts

```diff
@@ -73,6 +73,26 @@ export class Windows {
     this.__astNode = expression;
   }
 
+  /**
+   * Produces windows for each activity present in the plan and belonging to one of the activity types passed
+   *
+   * @param activityTypes the activity types
+   */
+  public static duringAll<A extends Gen.ActivityType>(list: Gen.ActivityType[]) : Windows {
```

Comment:
Thoughts on maybe calling this `duringAny`?

Reply:
Or `duringOr`, depending on the order we merge this and #407.

Reply:
my two cents is, I'd prefer to just make `during` take varargs

Also in the constraints eDSL the convention is to capitalize static functions (`Windows.During(...)`). I don't know if that's standard, but at least we're consistent.

Reply:
3 comments, 3 different ideas :) I'll go with `during` and the capitalization.

Reply:
`Windows.During` and `Windows.DuringAny`. And varargs.